tests/test_clients_GcpClient.py

```diff
@@ -1,5 +1,6 @@
 import os
 import unittest.mock
+from tests.utils.utilities import create_start_patcher, stop_all_patchers
```

Comment:
I see that you have imported `create_start_patcher` and `stop_all_patchers`, but have not defined anywhere. Is is like a built-in function? Then, couldn't understand why it was defined earlier.

Reply:
ok.. got it .. thank you!